java/src/jmri/configurexml/BlockManagerXml.java

```diff
@@ -302,6 +305,9 @@ private void loadBlock(Element element, boolean contentsFlag) throws JmriConfigu
             }
             block.setPermissiveWorking(permissive);
         }
+        if (element.getChild("ghost") != null) {
+            block.setIsGhost(element.getChild("ghost").getText().equals("yes") ? true : false );
```

Comment:
A minor issue :

Since equals() returns a boolean value, you don't need true : false.

So the code can be simplified with:

block.setIsGhost(element.getChild("ghost").getText().equals("yes") );